File: app/src/main/java/com/novoda/frankboylan/ncu/nodes/Metadata.java
package com.novoda.frankboylan.ncu.nodes;

class Metadata {
    
    private String title;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    @Override
    public String toString() {
        return "Metadata{" +
                "title='" + title + '\'' +
                '}';
    }
}

File: app/src/main/java/com/novoda/frankboylan/ncu/datamodel/Metadata.kt
package com.novoda.frankboylan.ncu.datamodel

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "metadata")
data class Metadata(

        @PrimaryKey
        @ColumnInfo(name = "department-code")
        @SerializedName("department-code")
        val departmentCode: String,

        @ColumnInfo(name = "journey-title")
        @SerializedName("title")
        val title: String,

        @ColumnInfo(name = "last-updated-timestamp")
        @SerializedName("last-updated-timestamp")
        val lastUpdatedTimestamp: Int,

        @ColumnInfo(name = "colour-hex")
        @SerializedName("colour-hex")
        val hexColourCode: String
)
